pkg/standalone-utils/contracts/relayer/BaseRelayerLibrary.sol

```diff
@@ -157,14 +166,8 @@ contract BaseRelayerLibrary is IBaseRelayerLibrary {
      * (see `_isTemporaryChainedReference` function).
      */
     function _getChainedReferenceValue(uint256 ref) internal override returns (uint256 value) {
-        bytes32 slot = _getStorageSlot(ref);
-
-        // Since we do manual calculation of storage slots, it is easier (and cheaper) to rely on internal assembly to
-        // access it.
-        // solhint-disable-next-line no-inline-assembly
-        assembly {
-            value := sload(slot)
-        }
+        bytes32 slot;
+        (slot, value) = _peekChainedReferenceValue(ref);
```

Comment:
I'd delete the named return value, declare both slot and value here, and explicitly return value at the end